.travis.yml

```diff
@@ -145,7 +145,7 @@ cache:
   directories:
     - $HOME/Library/Caches/Homebrew
 before_install:
-   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; brew install bash ; fi
+   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew upgrade bash; fi
```

Comment:
```suggestion
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export HOMEBREW_NO_INSTALL_CLEANUP=1 && brew update && brew install --verbose --debug bash; fi
```
Can you add the following:
* ` export HOMEBREW_NO_INSTALL_CLEANUP=1` the vm is anyway thrown away
* use `install` instead of `upgrade` because there should be no homebrew-based `bash` package installed
* use `brew install --verbose --debug bash` to see what's happening there.

Currently, it's not obvious at all maybe that helps.

Independent of that, I don't know why building a pull request is different than building a branch.
But let's find out.

Reply:
Thanks, Martin, I've also just seen at https://docs.brew.sh/FAQ the `HOMEBREW_NO_INSTALL_CLEANUP` option, sounds promising.  Good point about `bash install`.